CONTRIBUTING.md

```diff
@@ -20,9 +20,6 @@ To run tests scripts, you have to install:
 - [asdf](https://asdf-vm.com/guide/getting-started.html)
 - [starknet-devnet](https://0xspaceshard.github.io/starknet-devnet/docs/intro)
 
-> ⚠️ Make sure you run `./scripts/prepare_for_tests.sh` after setting up the development environment, otherwise the
-> tests will fail.
-
```

Comment:
Is this no longer necessary? Did it only setup fork?

Reply:
I checked and `test_cast.sh` runs `prepare_for_tests.sh` so I guess it's no longer needed

Reply:
I think we could still leave it here? In case someone wants to run tests with `cargo test`?

Reply:
Sure, will revert it